src/shaders.c

```diff
@@ -55,3 +55,80 @@ const char * const distortion_frag =
     "//Black edges off the texture\n"
     "gl_FragColor = ((tc_g.x < 0.0) || (tc_g.x > 1.0) || (tc_g.y < 0.0) || (tc_g.y > 1.0)) ? vec4(0.0, 0.0, 0.0, 1.0) : vec4(red, green, blue, 1.0);\n"
 "}";
+
+// OpenGL 3.0 compatible shaders, because not all platforms allow you to use OpenGL 1/2 style stuff anymore when enabling OpenGL 3...
+// You need an empty vao for this, no need to load up vertex buffer. This is assuming you limit left/right with glViewport. 
+// glBindVertexArray(vao);
+// glDrawArrays(GL_TRIANGLES, 0, 3 * 2);
+// glBindVertexArray(0);
```

Comment:
I'll take it out for now, it really needs an example application. Problem is that OpenGL 3 setup is a bit more involving then the simple OpenGL 1 example that exist right now